### PR TITLE
Add chapters from 1 to 3

### PR DESCRIPTION
**Pull request by SergioGueruiz**

### CODE REVIEW

File: Chapter3/block.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+=end
+
+#4.times { |i| puts "Count: #{i+1}" }
+
+#upto and downto methods
+5.downto(0) {|i| puts "Countdown: #{i}"}
+1.upto(5) {|i| puts "Count: #{i}"}
+
+#step method
+0.step(10, 2) {|i| puts"Step 2: #{i}"}
```
Comment (mauriciosabogal) [inline Chapter3/block.rb:26]: What is the difference between these methods: `times`, `downto`, `upto` and `step`?

File: Chapter3/block.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+=begin
+3.times do
+  puts "hello".capitalize
+  puts "world".capitalize
+  puts
+end #Multiple-line block
+=end
+
+=begin
+3.times do |i|
```
Comment (mauriciosabogal) [inline Chapter3/block.rb:13]: What is this block doing?

```ruby
3.times do |i|
  puts "Count: #{i+1}"
  puts "Next"
end
```

Comment (SergioGueruiz) [reply Chapter3/block.rb:13]: Is a loop that prints Count: + number of loop  and then in a new line prints Next, when it ends, adds 1 to iteration that's represent by  `i`  (in this case start in 0 and finish in 2)  and prints again but with another value  in `i`

File: Chapter3/block.rb (hunk 1 above)
Comment (SergioGueruiz) [reply Chapter3/block.rb:26]: `times:` The number that calls the method is the limit of iterations (start in 0) in each iteration runs the block and adds one to iteration to reach (number-1).

`downto:` The number that calls the  method will be subtracted 1 with each iteration until it's equal to the value that's in the parentheses, in each iteration runs the block/line.

`upto:` This is the opposite of `downto`, with each iteration adds 1 to the number until it's equal to the value that's in the parentheses, in each iteration runs the block/line.

`step:` Is like `range` in Python, the `number` that calls the method is the start, the first argument is the limit, the last argument is the value that adds to `number`, with each step/add runs the block and stops when the number is equal or greater than limit.

**Review by mauriciosabogal — APPROVED** [review]

**Pull request merged**